gui/qt/main_window.py

```diff
@@ -1324,6 +1324,8 @@ def output_for_opreturn_stringdata(op_return):
     def output_for_opreturn_rawhex(op_return):
         if not isinstance(op_return, str):
             raise OPReturnError('OP_RETURN parameter needs to be of type str!')
+        if op_return == 'empty':
```

Comment:
I get why you would set `op_return_raw` to `'empty'` (which is an invalid value for op_return_raw and would result in a form error).

But why check `op_return` == `'empty'` here?  I don't get it?

Also if I send you a payment with `&op_return=empty` it ends up being the empty string.  What if I wanted it to NOT be the empty string but really be the string `'empty'`? ;)

Reply:
Maybe you mean this conditional to check `if op_return_raw == 'empty':` and not `if op_return == 'empty':`?

Reply:
This function in particular only gets called when the 'raw' checkbox is ticked, so here `op_return` *is* the raw data. Just badly named variable name, which I should rename to `op_return_raw`. :-D

To clarify -- 'empty' only gets converted to blank if the 'raw' checkbox is ticked.

Reply:
Yeah I get it -- my bad.  Yeah that's fine then.